src/events/tab_event_handler.ts

```diff
@@ -43,8 +43,10 @@ export class tabEventHandler {
 		if (!repoId) return
 		// Get list of current tabs
 		const openTabs = vscode.window.tabGroups.all;
-		const tabs: ITabFile[] = openTabs.flatMap(tabGroup => 
+	try {	
+			const tabs: ITabFile[] = openTabs.flatMap(tabGroup => 
 			tabGroup.tabs.map(tab => {
+				if (!tab.input) return null;
```

Comment:
Can we improve this as 
```
if (!tab.input || !tab.input.uri || !tab.input.uri.path)  return null;
```

Will handle all properties.
Also, the error was accessing "path" property, not "input.uri" I believe

Reply:
Good idea